src/static/chemiscope.css

```diff
@@ -37,14 +37,120 @@
     z-index: 10;
 }
 
-.chsp-open-viewer-settings {
+.chsp-open-viewer-settings{
     position: absolute;
     top: 5px;
     right: 5px;
     /* Place on top of jsmol applet */
     z-index: 10;
 }
 
+.chsp-circle-button {
```

Comment:
Is this used for the color circle on the viewer side? If so, how about naming it `.chsp-structure-selected-marker`?